library/packages/src/modules/Package.rb

```diff
@@ -221,6 +222,18 @@ def reset
       @removed_packages.clear
     end
 
+    # Tries to find a package according to the pattern
+    #
+    # @param pattern [String] a pattern to match, no escaping done
+    # @return list of matching package names
+    def by_pattern(pattern = "")
+      init_packager
+      Y2Packager::Resolvable.find(provides_regexp: "^#{pattern}$")
```

Comment:
Maybe a comment noting that [Y2Packager::Resolvable.find](https://www.rubydoc.info/github/yast/yast-yast2/master/Y2Packager/Resolvable.find) takes POSIX EREs as opposed to Ruby Regexp below?

Also, its docs says to specify the used attrs for performance reasons
```suggestion
      Y2Packager::Resolvable.find(provides_regexp: "^#{pattern}$", [:name])
```

Reply:
thanks for the tip